rc/tools/man.kak

```diff
@@ -63,8 +63,11 @@ define-command -hidden -params ..3 man-impl %{ evaluate-commands %sh{
 
 define-command -params ..1 \
     -shell-script-candidates %{
-        find /usr/share/man/ $(printf %s "${MANPATH}" |
-            sed 's/:/ /') -name '*.[1-8]*' |
+        : "${MANPATH:="$(manpath)"}"
+        : "${MANPATH:=/usr/share/man}"
+        set -f
+        IFS=:
+        find $MANPATH -name '*.[1-8]*' |
```

Comment:
> manpath will find the corresponding share/man directories for each entry in PATH.

where is this documented?
I can see the `PATH` behavior mentioned in [FreeBSD man](https://manpage.me/?q=manpath)
but not in man-db man